app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -300,10 +300,11 @@ private void initializeContacts() throws IOException {
         final File file = new File(this.contactsPath);
         if (!file.exists()) {
             LOGGER.warn("Contacts part of the database didn't exist");
-            final Writer writer = this.fileSystem.getWriter(this.contactsPath);
-            writer.write("[]");
-            writer.close();
-            LOGGER.info("Created the contacts part of the database");
+            try (Writer writer = this.fileSystem.getWriter(this.contactsPath)) {
+                writer.write("[]");
+                writer.close();
+                LOGGER.info("Created the contacts part of the database");
+            }
```

Comment:
No, I'm only using the try to make sure that the resources I allocate are released, not to catch errors